feat(crons): Add note on consecutive failure alerts (#6078)
getsentry/sentry-docs: src/docs/product/crons/alerts.mdx

Navigate to the alerts tab in Sentry and create a new issue alert. You should see the issue alert builder. From here, you can start configuring your cron monitor alert rule. First, select the project corresponding to your cron monitor. Then, in the `IF` section of the alert rule configuration, add a new filter. You'll want to select the filter labeled "The event's tags match {key} {match} {value}".

<Note>

> You can use the "Number of events in an issue is" trigger filter to only alert you if the monitor has failed multiple consecutive check-ins. For example, if your job is supposed to run every minute, you could use the trigger filter **Number of events in an issue is more than 10 in 15 minutes**, to be alerted only after it's failed 10 times in that timeframe.

Author: ```suggestion
You can use the "Number of events in an issue is" trigger filter to only alert you if the monitor has failed multiple consecutive check-ins. For example, if your job is supposed to run every minute, you could use the trigger filter **Number of events in an issue is more than 10 in 15 minutes**, to be alerted only after it's failed more than 10 times in that timeframe.
```

Author: @lizokm

Navigate to the alerts tab in Sentry and create a new issue alert. You should see the issue alert builder. From here, you can start configuring your cron monitor alert rule. First, select the project corresponding to your cron monitor. Then, in the `IF` section of the alert rule configuration, add a new filter. You'll want to select the filter labeled "The event's tags match {key} {match} {value}".

<Note>

You can use the "Number of events in an issue is" trigger filter to only alert you when the monitor has failed multiple consecutive check-ins.

> For example if you're job should run every minute you could use the trigger filter **Number of events in an issue is more than 10 in 15 minutes** to be alerted after it's failed 10 times in that window.

Reviewer: your*

> 
> You can use the "Number of events in an issue is" trigger filter to only alert you when the monitor has failed multiple consecutive check-ins.
> 
> For example if you're job should run every minute you could use the trigger filter **Number of events in an issue is more than 10 in 15 minutes** to be alerted after it's failed 10 times in that window.
> 

Reviewer 2: ```suggestion

You can use the "Number of events in an issue is" trigger filter to only alert you if the monitor has failed multiple consecutive check-ins. For example, if your job is supposed to run every minute, you could use the trigger filter **Number of events in an issue is more than 10 in 15 minutes**, to be alerted only after it's failed 10 times in that timeframe.

```
Per this rule, shouldn't it be if it's failed MORE than 10 times in that timeframe?? :)

Author: you're right yeah more
